apps/ponder-interop/ponder.schema.ts

```diff
@@ -52,3 +52,73 @@ export const relayedMessages = onchainTable(
     transactionHash: t.hex().notNull(),
   }),
 )
+
+export const gasTankGasProviders = onchainTable(
```

Comment:
since we support withdrawals, lets start with just eth_call the contract instead of indexing every deposit?

Would reduce the complexity